src/components/renderer/form/form-renderer.component.tsx

```diff
@@ -91,7 +91,16 @@ export const FormRenderer = ({ processorContext, initialValues, setIsLoadingForm
             />
           );
         }
-        return <PageRenderer page={page} />;
+        return (
+          <PageRenderer
+            key={page.label}
+            page={page}
+            isCollapsed={false}
+            onToggleCollapse={function (): void {
+              throw new Error('Function not implemented.');
+            }}
+          />
+        );
```

Comment:
From the look of things you had added this implementation because the state was being managed within this file and now that it has been moved to the `PageRenderer`, there's no need for the extra params